model/Core/Properties/processRational.md

```diff
@@ -1,6 +1,6 @@
 SPDX-License-Identifier: Community-Spec-1.0
 
-# processRationale
+# processRational
```

Comment:
"rationale" is a noun.

"rational" is an adjective (being rational)

I guess that in this context, we would like to have a noun (reason is a noun).

In either way, we may like to keep this consistent with `requirementsRationale` as well (renaming `requirementsRationale` to `requirementsRational` too or the other way around).

Reply:
Agree with Art.